forge/files/src/handle-file-urls.js

```diff
@@ -0,0 +1,29 @@
+const { protocol } = require('electron');
+const { fileURLToPath } = require('url');
+const path = require('path');
+const fs = require('fs');
+const { promisify } = require('util');
+
+const access = promisify(fs.access);
```

Comment:
Remind me again why we can't use [`fsPromises`](https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fspromises_access_path_mode)... Need to support Node < 10.x?

Reply:
Uh, yeah, we kinda recently dropped node 8 support, so that's why we were avoiding it before. However, this is a different case, because this code runs in the Electron main process, not the user's Node environment, so the `engines` in `package.json` isn't relevant here, just the version of Electron that the user is running.

But Electron has been on Node 10+ since 3.x, and I'm quite confident that there are other reasons this code would break on Electron <=2, not the least of which is the usage of `fileURLToPath` in this same file, which wasn't introduced until Node 10.12/Electron 5 😆 

I don't think we have an official policy on which versions of Electron we do and don't target, but since:
1. This is all blueprint-generated code that the user could modify/replace if they need to
2. We're in the midst of a major version bump

I think we're fine keeping the usage of `fileURLToPath` and using `fs.promises`. I'll make the change.

Reply:
I think we should maybe support Electron 5+ or whatever we had to update Electron Forge for?

Reply:
FWIW the [Electron team only supports the latest 3 stable branches](https://www.electronjs.org/docs/tutorial/support#supported-versions). So now that 9.0.0 is out I think that means versions older than 7.x are no longer supported. I am fine with 5+, but my gut would be to target actively supported versions.

Reply:
*grumble grumble* https://travis-ci.com/github/adopted-ember-addons/ember-electron/jobs/337264506#L610

Reply:
I would be fine with Electron 7+, and I would honestly also be fine with bumping node minimum to 12, but ember-cli supports 10 until April 2021.

Reply:
Yeah, this is kinda silly because `eslint` is warning us about code that runs in Electron, not the environment that the `engines` field is describing. So I don't think bumping our minimum required node version makes any sense because I don't think we use any `fs.promises` in code that actually runs in the user's node environment, so we'd be dropping support just to make the linter happy.

That being said, I do think it makes sense to get more specific about what Electron versions we support, but haven't put any thought into really how to accomplish that -- I guess it would just be documentation and then testing against all supported (major) versions?

Reply:
Yeah, I think we would mainly list in the documentation that we don't support earlier Electron versions, but that it may still work.